<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateEventPage</class>
 <widget class="QWidget" name="CreateEventPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>780</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="nameLayout">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Event Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameField"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="categoryLayout">
         <item>
          <widget class="QLabel" name="categoryLabel">
           <property name="text">
            <string>Category:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="categoryField"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="venueNameLayout">
         <item>
          <widget class="QLabel" name="venueNameLabel">
           <property name="text">
            <string>Venue Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="venueNameField"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="venueAddressLayout">
         <item>
          <widget class="QLabel" name="venueAddressLabel">
           <property name="text">
            <string>Venue Address:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="venueAddressField"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="venuePhoneLayout">
         <item>
          <widget class="QLabel" name="venuePhoneLabel">
           <property name="text">
            <string>Venue Phone Number:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="venuePhoneField"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="startTimeLabel">
           <property name="text">
            <string>Start:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="startTime">
           <property name="displayFormat">
            <string>M/d/yyyy h:mm AP</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="endTimeLabel">
           <property name="text">
            <string>End:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="endTime">
           <property name="displayFormat">
            <string>M/d/yyyy h:mm AP</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="descriptionLayout">
         <item>
          <widget class="QLabel" name="descriptionLabel">
           <property name="text">
            <string>Event Description:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="descriptionField"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonsLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="submitButton">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
